src/graphql/main/schema.graphql

```diff
@@ -317,6 +318,39 @@ type MobileVersions {
   minSupported: Int!
 }
 
+type PricePoint {
+  timestamp: Timestamp!
+  price: Price!
+}
+
+type Price {
+  base: SafeInt!
+  offset: Int!
+  currencyUnit: ExchangeCurrencyUnit!
+  formattedAmount: String!
+}
+
+"""
+The `SafeInt` scalar type represents non-fractional signed whole numeric values. SafeInt can represent values between -(2^53) + 1 and 2^53 - 1
+"""
+scalar SafeInt
+
+enum ExchangeCurrencyUnit {
+  BTCSAT
+  USDCENT
```

Comment:
This pr looks good to me. Wanted to highlight this for @bodymindarts, and see what he thinks about putting both the currency and unit into one string type like this.